### PR TITLE
Updates in the Backend Code

### LINKED ISSUE #12: Enhancement in backend
In this project's backend require() method is used to import the packages, instead we can use import statement as a good practice..

### PR DESCRIPTION
**Pull request by Muhammad-Owais-Warsi**
## Related Issue #12 

Closes #12 

## Description

Made the updates in the backend code -
1. Changed the require type to import 
2. Made the changes, so that after the deployment no previous repos should get lost

## Checklist 

- [x] My code adheres to the established style guidelines of the project.
- [  ] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -130,12 +131,24 @@ const addRepoToCheck = (repoURL) => {
...
-    sendGFIEmail(repoURL);
-    console.log(`Trying to send an email for ${repoURL}.`);
-  });
+const cronJob = new cron.CronJob('* * * * *', async () => {
+  
+  console.log('Checking for new GFIs...');
+  try {
+    if (repoToCheck.length == 0) {
+      let repoData = await Subscription.find({});
+      repoData = [...new Set(repoData.map(item => item.repoUrl))];
```
Comment (ItIsOHM) [inline backend/server.js]: This needs to be changed to `repoURL` instead of `repoUrl`

**Review by ItIsOHM — CHANGES_REQUESTED** [review]
Everything looks alright just check the review.

File: backend/server.js (hunk 1 above)
Comment (Muhammad-Owais-Warsi) [reply backend/server.js]: Ok, I'll change it

**Comment by Muhammad-Owais-Warsi** [comment]
Also , I want to know that, whenever I put any repo name it shows me this
Screenshot_2024_0220_170354

**Pull request merged**